app.rb

```diff
@@ -0,0 +1,50 @@
+require 'sinatra/base'
```

Comment:
Looking at your code I'm inspired on how to tackle the log-in and registration process.